float32 presa_x
float32 presa_y
float32 presa_theta
float32 objetivo_x
float32 objetivo_y
---
float32 lineal_presa
float32 angular_presa
float32 distancia
